pkg/policy/l4.go

```diff
@@ -1163,14 +1390,14 @@ func (l4 *L4DirectionPolicy) attach(ctx PolicyContext, l4Policy *L4Policy) redir
 // Otherwise, returns api.Allowed.
 //
 // Note: Only used for policy tracing
-func (l4 L4PolicyMap) containsAllL3L4(labels labels.LabelArray, ports []*models.Port) api.Decision {
-	if len(l4) == 0 {
+func (l4M *l4PolicyMap) containsAllL3L4(labels labels.LabelArray, ports []*models.Port) api.Decision {
```

Comment:
maybe comment why the LPM lookup is the correct thing to do here rather than the exact lookup?

Reply:
I renamed "Lookup" to "LongestPrefixMatch" instead.